.github/workflows/dev-build.yml

```diff
@@ -83,7 +83,7 @@ jobs:
   allocate-runners:
     name: Allocate runners
     if: ${{ github.repository == 'GreptimeTeam/greptimedb' }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Do we need to specify the ubuntu version here? @zyy17

Reply:
This one is only used for allocate actual builders I think